### ISSUE TITLE
[Mobile] Sign-in with Apple/Google

### ISSUE THREAD

**Issue by morning4coffe-dev** [unoplatform/uno.toolkit.ui#1264]
## What would you like to be added:

Based on https://github.com/unoplatform/uno/pull/18410, there was an internal discussion that it would be nice, if we added similar functionality directly in Toolkit/Wizard. 

## Why is this needed:

Would be a nice feature to have and it would make the development of many apps much easier.

## For which Platform:

- [ ] WebAssembly
- [x] Android
- [x] iOS
- [ ] macOS (AppKit)
- [ ] Mac Catalyst
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Linux Framebuffer
  - [ ] Tizen
- [ ] Windows

**Comment by kazo0** [comment]
> Is this toolkit or rather a Extensions Authentication feature 🤔

Yes, this would make most sense as part of Uno Extensions Authentication. Though you should be able to do this individually using the OIDC extensions I believe? 

Although we could have platform specifics for integrations directly with Android/iOS for Google/Apple sign-ins
